ui/shared/AddressInput.qml

```diff
@@ -7,9 +7,11 @@ import "../imports"
 Item {
     id: root
     property string validationError: "Error"
+    property string asyncValidationError: qsTr("ENS Username not found")
```

Comment:
At the moment, we let the outside world decide about what `validationError` is and then pass it down to `AddressInput`...
I considered doing the same for `asyncValidationError` but I ended up not going down that path due to following reasons:

1. The responsibility of addressInput is clearly defined. It won't make sense to give it any error message that isn't related to the core functionality fo this component
2. Most likely, we want the component to work as a drop-in component, so that it just-works™ without giving it all the pieces for its core functionality.